### ISSUE TITLE
Standalone doc is not correct for CLion

### ISSUE THREAD

**Issue by david-29** [TheLartians/ModernCppStarter#121]
The build directory in CLion is 'cmake-build-debug', not 'build'.

Or you can right-click Standalone's CMakeLists.txt file and select 'Load CMake Project'. Then Standalone becomes available to Run or Debug.

**Comment by TheLartians** [comment]
Yeah, the actual executable location is dependent on the CMake generator used. I currently assume Makefiles, but perhaps we should specify that better in the docs!
